activesupport/lib/active_support/claims.rb

```diff
@@ -0,0 +1,46 @@
+require 'active_support/time'
+
+module ActiveSupport  
+  class Claims
+    class InvalidClaims < StandardError; end
+    class ExpiredClaims < StandardError; end
+    
+    attr_reader :payload, :purpose, :expires
+
+    def initialize(options)
```

Comment:
Actually, there are some more things to figure out like what should be the expiry to be signed inside jwts when expiry is not mentioned (i.e for browser session expiries). So, default purpose can be set, payload can be made mandatory but I'm not sure what do we do about expiry? What do you think @kaspth ?